skcosmo/linear_model/_ridge.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+
+from sklearn.model_selection import KFold
+from sklearn.base import RegressorMixin, MultiOutputMixin
+from sklearn.metrics import check_scoring
+
+from joblib import Parallel, delayed
+
+
+class RidgeRegression2FoldCV(MultiOutputMixin, RegressorMixin):
```

Comment:
no it cant be done, because CV only spits out train and test set, here we actually change the way the computation of the weights is done within the CV. That is why there exist RidgeCV in sklearn, they could you just do a pipeline with RidgeRegression or RidgeClassification if they would use standard CV, but RidgeCV has a specific optimization for the case where cv=None, that is why they have an additional class.